### PR TITLE
Add additional function skus

### PR DESCRIPTION
**Pull request by likelymichael**
closes: https://linear.app/massdriver/issue/ORC-193/expand-sku-list-for-azure-app-templates

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -117,3 +117,6 @@ schema-*.json
 
 # Infracost
 *.infracost
+
+# Azurite
```
Comment (WillBeebe) [inline .gitignore:121]: what's Azurite mean?

File: azure-function-app/massdriver.yaml [hunk 2]
```diff
@@ -153,13 +153,29 @@ params:
...
+            - title: 1 vCPU, 1.75GB RAM, scales up to 3 instances
+              const: B1
+            - title: 2 vCPU, 3.5GB RAM, scales up to 3 instances
+              const: B2
+            - title: 4 vCPU, 7GB RAM, scales up to 3 instances
+              const: B3
+            - title: 1 vCPU, 1.75GB RAM, scales up to 10 instances
               const: S1
-            - title: 2 vCPU 8GB RAM
+            - title: 2 vCPU, 3.5GB RAM, scales up to 10 instances
```
Comment (WillBeebe) [inline azure-function-app/massdriver.yaml:164]: Did you provision any of these? I know we've have some sku issues lately and I'm hesitant to take these in blindly ya know. Could I get a Terraform run of a few of them please?

**Review by WillBeebe — CHANGES_REQUESTED** [review]
Just a few comments, looking good!

File: .gitignore (hunk 1 above)
Comment (likelymichael) [reply .gitignore:121]: Azurite is a dev tool for azure storage account emulation

File: azure-function-app/massdriver.yaml (hunk 2 above)
Comment (likelymichael) [reply azure-function-app/massdriver.yaml:164]: Will test as soon as mdxc issue is fixed, currently preventing apps from deploying

**Comment by likelymichael** [comment]
This should be done via satellite if possible

**Pull request closed without merging**